src/transactionsFactories/transferTransactionsFactory.ts

```diff
@@ -89,27 +155,184 @@ export class NextTransferTransactionsFactory {
         }).build();
     }
 
+    createEGLDTransfer(args: {
+        nonce?: INonce;
+        value: ITransactionValue;
+        receiver: IAddress;
+        sender: IAddress;
+        gasPrice?: IGasPrice;
+        gasLimit?: IGasLimit;
+        data?: ITransactionPayload;
+        chainID: IChainID;
+    }) {
+        if (!this.isGasEstimatorDefined()) {
+            throw new Err("`gasEstimator` is not defined. Instantiate the factory using the gasEstimator.");
```

Comment:
Alter the message perhaps to recommend the new functions, instead.

_"It seems that you are calling the legacy functions to create a transfer transaction. If this is your intent, then pass a gas estimator in the constructor. Or, instead, use the new, recommended ..."._